.pylintrc

```diff
@@ -32,13 +32,13 @@ good-names=e,_,d,f,i,l,p,w,fn,fs,id,pc,sc,zf,setUp,tearDown,longMessage,maxDiff
 dummy-variables-rgx=_|unused_*
 
 # Regular expression which should match good variable names
-variable-rgx=[a-z_][a-z0-9_]*$
+variable-rgx=^[a-z_][a-z0-9]+(_[a-z0-9]+)*$
```

Comment:
You need to rethink this regex, this is incorrect.

Reply:
Now it can handle single character variable names too. Variable name must start with an alphabet. If `_` is the first character then the second literal must be an alphabet. I hope the regex looks correct now.